src/main/java/mate/academy/springbootwebgreqit/model/User.java

```diff
@@ -39,14 +39,17 @@ public class User implements UserDetails {
     @OneToOne(cascade = CascadeType.ALL)
     private ShoppingCart shoppingCart;
 
-    @ManyToMany(fetch = FetchType.EAGER)
+    @ManyToMany(fetch = FetchType.LAZY)
     @JoinTable(
             name = "users_roles",
             joinColumns = @JoinColumn(name = "user_id"),
             inverseJoinColumns = @JoinColumn(name = "role_id")
     )
     private Set<Role> roles = new HashSet<>();
 
+    @OneToMany(mappedBy = "user", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+    private Set<Order> orders = new HashSet<>();
+
     @Override
     public Collection<? extends GrantedAuthority> getAuthorities() {
         if (email.startsWith("alicewe")) {
```

Comment:
what is the hardcoded "alicewe" ?
```suggestion
        return roles;
```